### PR TITLE
support set cloudcore config file path in keadm init

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: manifests/charts/cloudcore/templates/configmap_cloudcore.yaml [hunk 1]
```diff
@@ -6,6 +6,12 @@ metadata:
   labels: {{- toYaml . | nindent 4 }}
   {{- end }}
 data:
+  {{- if (ne .Values.cloudCore.configFile "") }}
+  {{- $files := .Files }}
+  {{- $config := .Values.cloudCore.configFile -}}
+  cloudcore.yaml: |
+      {{ $files.Get "$config" }}
```
Comment (gy95) [inline manifests/charts/cloudcore/templates/configmap_cloudcore.yaml:13]: @zhu733756  hi, do you know how to access a file outside `charts` directory, e.g. /etc/kubeedge/config/cloudcore.yaml. I found files.Get can only get the content of files in charts directory.

Comment (zhu733756) [reply manifests/charts/cloudcore/templates/configmap_cloudcore.yaml:13]: I see a function like here in https://helm.sh/docs/chart_template_guide/accessing_files/  .
The page says that "Then we print each file name ({{ . }}: |-) followed by the contents of the file {{ $files.Get . }}."
Can you try it?

Comment (gy95) [reply manifests/charts/cloudcore/templates/configmap_cloudcore.yaml:13]: we cannot access the files outside chart directory , by asking in helm slack, the answer is below
`
That doesnt work by design. Imagine you give this chart to your colleage. If they dont have the file in the same place, the chart is broken. It is probabably also a security risk. Imagine some third party chart goes out and fetches your /etc/password file.
`

Comment (zhu733756) [reply manifests/charts/cloudcore/templates/configmap_cloudcore.yaml:13]: Got it. I think they are correct.

**Comment by gy95** [comment]
cannot be implemented, ref the above comments
/close

**Pull request closed without merging**